app/code/core/Mage/Adminhtml/Block/Newsletter/Subscriber/Grid/Filter/Website.php

```diff
@@ -27,18 +27,28 @@
  */
 class Mage_Adminhtml_Block_Newsletter_Subscriber_Grid_Filter_Website extends Mage_Adminhtml_Block_Widget_Grid_Column_Filter_Select
 {
+    /**
+     * @var Mage_Core_Model_Resource_Website_Collection
+     */
     protected $_websiteCollection = null;
 
+    /**
+     * @return array|array[]
```

Comment:
Sorry, I see something new, `array[]`, what does it mean? Looks like array of array. Is it not enough with just one: `@return array` or `@return array[]`?